packages/models/src/errors.ts

```diff
@@ -422,3 +423,11 @@ export function invalidKey(
     title: "Invalid Key",
   });
 }
+
+export function invalidKeyLength(key: string): ApiError<CommonErrorCodes> {
+  return new ApiError({
+    detail: `Invalid length for the key ${key}, must be 2048 bites`,
+    code: "invalidKeyLength",
+    title: "Invalid Key length",
+  });
+}
```

Comment:
I would avoid printing the entire key since it can be very long.

```suggestion
function invalidKeyLength(
  length: number | undefined,
  minLength: number
): ApiError<CommonErrorCodes> {
  return new ApiError({
    detail: `Invalid RSA key length: ${length} bits. It must be at least ${minLength}`,
    code: "invalidKeyLength",
    title: "Invalid Key length",
  });
}
```